arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/Schedule.kt

```diff
@@ -217,6 +221,25 @@ public sealed class Schedule<Input, Output> {
   public suspend fun repeatOrElse(fa: suspend () -> Input, orElse: suspend (Throwable, Output?) -> Output): Output =
     repeatOrElseEither(fa, orElse).fold(::identity, ::identity)
 
+  public abstract suspend fun <C> repeatAsFlow(
```

Comment:
`repeatOrElseEitherAsFlow` ?

Can the signature also be `fa: suspend () -> Input, orElse: suspend (Throwable, Output?) -> C` like `repeatOrElseEither`?

Reply:
That Output parameter may not be as important, as the previous output is in the Flow. I don't know if we want to support this special functionality

Reply:
If it is to preserves what the api expose with ‘repeat { }’ I can add it ☺️

Reply:
I would still add it. The signature from `repeat` exposes the most powerful behavior, since you can also easily ignore the parameter.